explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -40,13 +40,15 @@ import react.semanticui.sizes._
 import scala.concurrent.duration._
 
 final case class TargetTabContents(
-  userId:                ViewOpt[User.Id],
-  focused:               View[Option[Focused]],
-  searching:             View[Set[Target.Id]],
-  targetViewExpandedIds: View[TargetViewExpandedIds],
-  size:                  ResizeDetector.Dimensions
+  userId:      ViewOpt[User.Id],
+  focused:     View[Option[Focused]],
+  searching:   View[Set[Target.Id]],
+  expandedIds: View[ExpandedIds],
+  size:        ResizeDetector.Dimensions
 ) extends ReactProps[TargetTabContents](TargetTabContents.component) {
-  def isTargetSelected: Boolean = focused.get.isDefined
+  def isTargetSelected: Boolean = focused.get.collect { case Focused.FocusedTarget(_) =>
```

Comment:
Can this be done with `exist`?

Reply:
Everything I could think of that used `exist` was a lot less concise because you need total function. It can be done, though, of course.

Reply:
Ah right, I thought exist could take a partial